_sass/blocks/_revenues_subpage_intro.scss

```diff
@@ -3,32 +3,60 @@
 // This is the first block on a revenues subpage
 //
 // Styleguide blocks.revenues_subpage_intro
+.revenues_subpage-intro_wrapper {
+  border-top: 1px solid $mid-gray;
+}
 
 .revenues_subpage-intro {
-  padding-top: $base-padding;
+  padding-top: $base-padding-extra;
 
   h1 {
-    @include heading(h1);
-    padding-bottom: ($base-padding / 2);
+    line-height: 2rem;
+    margin-bottom: 0.7rem;
   }
 
-  p {
-    padding-bottom: $base-padding;
-  }
+}
+
+.revenues_subpage-intro_text{
+
+  @include heading('para-lg');
+
+  padding-bottom: $base-padding;
+}
+
+.revenues_subpage-breadcrumb {
+  color: $dark-gray;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets